src/parse-jobs.js

```diff
@@ -3,106 +3,86 @@ const cheerio = require('cheerio');
+        cultureAndValuesRating: (jsonCompanyInfo.initialState.jlData.overview.ratings
+            && jsonCompanyInfo.initialState.jlData.overview.ratings > 0)
+            ? jsonCompanyInfo.initialState.jlData.overview.ratings.cultureAndValuesRating : null,
+        workLifeBalanceRating: (jsonCompanyInfo.initialState.jlData.overview.ratings
+            && jsonCompanyInfo.initialState.jlData.overview.ratings > 0)
+            ? jsonCompanyInfo.initialState.jlData.overview.ratings.workLifeBalanceRating : null,
+        revenue: jsonCompanyInfo.initialState?.jlData?.overview?.revenue ?? null,
+        size: jsonCompanyInfo.initialState?.jlData?.overview?.size ?? null,
+        type: jsonCompanyInfo.initialState?.jlData?.overview?.type ?? null,
+        website: jsonCompanyInfo.initialState?.jlData?.overview?.website ?? null,
+        yearFounded: jsonCompanyInfo.initialState?.jlData?.overview?.yearFounded ?? null,
+    };
 
-            log.info(`Saving details for job listing id ${request.uniqueKey}`);
-            // unified output based on list item and sub page
-            await Apify.pushData({
-                ...updatedItem,
-                url: json.url,
-                salary: json.estimatedSalary,
-                jobLocation: json.jobLocation,
-                companyDetails: { ...json.hiringOrganization, ...moreDetails },
-                jobDetails: clearDetails,
-                datePosted: json.datePosted,
-            });
-        },
-        /*
-        handleFailedRequestFunction: async ({ request }) => {
-            await Apify.pushData({
-                '#isFailed': true,
-                '#debug': Apify.utils.createRequestDebugInfo(request),
-            });
+    log.info(`Saving details for job listing id ${request.uniqueKey}`);
+    // SAVING FINAL DATA
+    await Apify.pushData({
+        ...item,
+        url: `https://www.glassdoor.com${jsonCompanyInfo.initialState.parsedRequest.url}`,
+        salary: {
+            min: (jsonCompanyInfo.initialState.jlData.header.payPercentile10 && jsonCompanyInfo.initialState.jlData.header.payPercentile10 > 0)
+                ? jsonCompanyInfo.initialState.jlData.header.payPercentile10 : null,
+            max: (jsonCompanyInfo.initialState.jlData.header.payPercentile90 && jsonCompanyInfo.initialState.jlData.header.payPercentile90 > 0)
+                ? jsonCompanyInfo.initialState.jlData.header.payPercentile90 : null,
+            payPeriod: jsonCompanyInfo.initialState.jlData.header.payPeriod,
+            currency: jsonCompanyInfo.initialState.jlData.header.payCurrency,
+            source: jsonCompanyInfo.initialState.jlData.header.salarySource,
         },
-        */
+        jobLocation: jsonCompanyInfo.initialState.jlData.header.locationName,
+        companyDetails: { ...moreDetails },
+        jobDetails: clearDetails,
+        datePosted: jsonCompanyInfo.initialState.jlData.header.posted,
```

Comment:
I would convert this to a ISO time, using `new Date(jsonCompanyInfo.initialState.jlData.header.posted)`